### PR TITLE
Add machine status check in test playbook

### PR DESCRIPTION
**Pull request by maelk**
Verify that machines are in Running phase.

### CODE REVIEW

File: vm-setup/roles/v1aX_integration_test/tasks/verify.yml [hunk 1]
```diff
@@ -17,6 +17,16 @@
     delay: 20
     until: provisioned_bmh.stdout_lines ==  ["node-0", "node-1"]
 
+  - name: Wait until both machines become running.
+    shell: |
+        kubectl get machines -n metal3 -o json | jq -r '.items[]
+        | select (.status.Phase == "Running")
```
Comment (fmuyassarov) [inline vm-setup/roles/v1aX_integration_test/tasks/verify.yml]: I think it should be small ''p'' not capital in `.status.Phase`

Comment (maelk) [reply vm-setup/roles/v1aX_integration_test/tasks/verify.yml]: good catch!

**Pull request merged**